extensions/wikia/TemplateDraft/TemplateDraft.i18n.php

```diff
@@ -100,6 +100,8 @@
 	'templatedraft-module-view-parent' => 'Wikitext der übergeordneten Vorlage ansehen',
 	'templatedraft-module-help' => 'Hilfeseite für das Infobox-Format ansehen',
 	'templatedraft-invalid-template' => 'Ungültiger Vorlagenname angegeben.',
+	'templatedraft-module-approve-protected' => 'Diese Vorlage ist geschützt. Wende dich bitte an einen [[Special: ListAdmins| Administrator]], damit er diesen Entwurf freischalten kann.',
```

Comment:
Probably worth fixing up the spacing in the links i.e. `[[Special: ListAdmins| Administrator]]` -> `[[Special:ListAdmins|Administrator]]`. Though it won't break anything, so not too important.